docs/controls/risk_assessment/ra-05.md

```diff
@@ -0,0 +1,31 @@
+# Risk Assessment
+## RA-05 - VULNERABILITY SCANNING
+
+The organization:  
+a. Scans for vulnerabilities in the information system and hosted applications [Assignment: organization-defined frequency and/or randomly in accordance with organization-defined process] and when new vulnerabilities potentially affecting the system/applications are identified and reported;  
```

Comment:
Looks like there is a typo on the word "Assignment" (need to remove the "[")

Reply:
removed the "[" from places in the implementation statement.